packages/gestalt/src/ComboBox.js

```diff
@@ -144,6 +144,10 @@ type Props = {|
    * List of tags to display in the component. See [tags](https://gestalt.pinterest.systems/combobox#Tags) variant to learn more.
    */
   tags?: $ReadOnlyArray<Element<typeof Tag>>,
+  /**
+   * An object representing the zIndex value of the ComboBox list box. Learn more about [zIndex classes](https://gestalt.pinterest.systems/zindex_classes)
+   */
+  zIndex?: {| index(): number |},
```

Comment:
You can just use Indexable here (as we do with all other zIndex references)

Reply:
DONE